src/main/groovy/org/pih/warehouse/inventory/CycleCountDto.groovy

```diff
@@ -0,0 +1,20 @@
+package org.pih.warehouse.inventory
+
+class CycleCountDto {
+
+    String id
+
+    String status
+
+    List<CycleCountItemBasicDto> cycleCountItems
+
+    static CycleCountDto createFromCycleCountItems(List<CycleCountItem> items) {
+        // Cycle count items have the same cycle count associated, so we can just look at first item in order to get the cycle count properties
+        CycleCountItem item = items?.first()
```

Comment:
this is one of the "tricky" parts I mentioned above. If I had an access to the list of the items from the cycle count domain, I wouldn't have to do that.

Reply:
eventually we can check if every item has the same cycle count (in case in the future this method would be used for "mixed" cycle count items (with different cycle count associated) and throw an exception.
Now, it can't happen, so let me know your thoughts.